### DISCUSSION TITLE
Need assistance in setting everything up

### DISCUSSION THREAD

**Discussion by Andre601** [mmistakes/minimal-mistakes#2783] [Q&A]
Hello everyone.

I want to help someone to switch from their static html pages to a Jekyll hosted one and found this interesting theme to use.
Thing is, that I have 0 idea how to properly start.
I cloned the [MM Remote theme starter](https://github.com/mmistakes/mm-github-pages-starter) and setup an action that would automatically push changes from the folder of a specfic branch to another one. But now when I quickly check the page do I only have a blank page showing the frontmatter of the default index.html

![image](https://user-images.githubusercontent.com/11576465/104790625-0a09ec00-5798-11eb-9fe2-9a46074d0829.png)

I really have no clue on how to continue and f.e. update/change pages to make them look good (Or where they are saved under to begin with).

The repository in question is https://github.com/Andre601/5zig-reborn.github.io and the master branch is used for the live page.

I already tried the `_layout` option with the `home.html` file but it doesn't seem to work.
Do I need to essentially create all the files in the `_layouts` folder of this repo to make it work?

Appreciate any help to make this work. 

P.S.: I want to use the `dark` theme

**Comment by Andre601** [comment]
I still would appreciate help with getting this started. The quick-start template, while nice, lacks certain features/examples I personally want to use.

**Reply by tholonia** [reply]
<BUMP>  The quick start also did not work for me.  I get 

```
 Liquid Exception: No repo name found. Specify using PAGES_REPO_NWO environment variables, 'repository' in your configuration, or set up an 'origin' git remote pointing to your github.com repository. in /_layouts/default.html
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    No repo name found. Specify using PAGES_REPO_NWO environment variables, 'repository' in your configuration, or set up an 'origin' git remote pointing to your github.com repository.

```

I have tried all the recommended installation methods, and they all give the same error.  I have also tried adding repositories, remote themes, and local themes :/  I am new to docker/Jekyll (but I have managed to get other themes running), so if anyone does reply to this, please speak in beginner-ese.
